### PR TITLE
WIP on tests, bugfixes, additional documentation and specs

### PR DESCRIPTION
**Pull request by DimitrijeDimitrijevic**

### CODE REVIEW

File: lib/excalt/vcard/helpers.ex [hunk 1]
```diff
@@ -0,0 +1,3 @@
+defmodule Excalt.Vcard.Helpers do
+
```
Comment (swerter) [inline lib/excalt/vcard/helpers.ex:8]: What do you want to put here? If nothing for the moment I'd suggest to remove the file.

File: test/excalt/vcard/parser_test.exs [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        """
+        |> Excalt.Vcard.Parser.parse_addressbooks()
+
+      {:ok, response} = response
+
+      expected = [
+        %Excalt.Vcard.Addressbook{
+          name: "Family",
+          description: nil,
+          url: "/addressbooks/ccdav@dimitrijedimitrijevic.com/family/",
```
Comment (swerter) [inline test/excalt/vcard/parser_test.exs]: I would remove real email addresses.

**Review by swerter — CHANGES_REQUESTED** [review]
Looks good, thanks. Just a few small changes.

File: lib/excalt/vcard/helpers.ex (hunk 1 above)
Comment (DimitrijeDimitrijevic) [reply lib/excalt/vcard/helpers.ex:8]: Populated the file with auth function.

File: test/excalt/vcard/parser_test.exs (hunk 2 above)
Comment (DimitrijeDimitrijevic) [reply test/excalt/vcard/parser_test.exs]: Agree, changed to random email.

**Pull request merged**